tests/test_rhiza/test_readme.py

```
@@ -40,7 +40,7 @@ def test_readme_runs(logger, root):
     # Trust boundary: we execute Python snippets sourced from README.md in this repo.
     # The README is part of the trusted repository content and reviewed in PRs.
     logger.debug("Executing README code via %s -c ...", sys.executable)
-    result = subprocess.run([sys.executable, "-c", code], capture_output=True, text=True, cwd=root)  # noqa: S603
+    result = subprocess.run([sys.executable, "-c", code], capture_output=True, text=True, cwd=root)
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)